arraycontext/impl/pyopencl/__init__.py

```diff
@@ -186,6 +187,9 @@ def call_loopy(self, t_unit, **kwargs):
             if len(wait_event_queue) > self._wait_event_queue_length:
                 wait_event_queue.pop(0).wait()
 
+        # Add the returned event to the dictionary
+        result["evt"] = evt
```

Comment:
This isn't ideal IMO. It could clash with existing names, and the event can already be fished out of the output arrays.

Reply:
Removed and changed grudge code to use an event from the output arrays.